tests/exclusive_scan.cpp

```diff
@@ -55,7 +55,7 @@ void exclusive_scan_gold(std::vector<T>& v, T init = 0,
 
 // test the gold computation against a known value
 TEST_F(ExclusiveScanAlgorithm, TestSTDExclusiveScan) {
-  std::vector<int> v = {5, 1, 6, 2, 6, 2, 5, 7};
+  std::vector<int> v    = {5, 1, 6,  2,  6,  2,  5,  7};
```

Comment:
@Ruyk I thought the same first and I understood the "visual computing" aspect of it with my Emacs and a fixed-space font. :-)